lib/templates/error.html

```diff
@@ -1091,31 +1102,34 @@ <h1 class="error-header">Dang! Looks like a {{errorType}}.</h1>
           <h2 class="error-message">{{location.file}}{{#if location.line}}:{{location.line}}{{/if}}</h2>
         {{/if}}
 
-        {{#if codeFrame}}
-          <pre><code class="javascript">{{codeFrame}}</code></pre>
-        {{/if}}
+          {{#if codeFrame}}
+            <pre><code class="javascript">{{#if hasAnsi}}{{{codeFrame}}}{{else}}{{codeFrame}}{{/if}}</code></pre>
```

Comment:
Is there a better way to do this?

Reply:
not that I can think of this looks good to me!